meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -15,6 +15,8 @@
 #  INTERNAL_DEVICE_BOOTLOADER_CONFIG      - name of the bootloader config for internal media boot
 #  INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH - relative path to internal boot partition where
 #                                               INTERNAL_DEVICE_BOOTLOADER_CONFIG will be copied to
+#  INTERNAL_DEVICE_BOOTLOADER_LEGACY_CONFIG_PATH - [Optional] same as INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH
+#                                                     but for legacy bootloaders
```

Comment:
I think you should indent all the other lines accordingly since you introduced a new variable with a longer name